elytron/src/main/java/org/wildfly/extension/elytron/FileSystemRealmDefinition.java

```diff
@@ -110,7 +119,26 @@
             .setAllowExpression(true)
             .build();
 
-    static final AttributeDefinition[] ATTRIBUTES = new AttributeDefinition[]{PATH, RELATIVE_TO, LEVELS, ENCODED, HASH_ENCODING, HASH_CHARSET};
+    static final SimpleAttributeDefinition CREDENTIAL_STORE =
+            new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.CREDENTIAL_STORE, ModelType.STRING, false)
+                    .setAllowExpression(true)
```

Comment:
Because this new attribute allows expressions, we'll also want to update https://github.com/wildfly/wildfly-core/blob/main/elytron/src/test/resources/org/wildfly/extension/elytron/elytron-expressions.xml to add a filesystem realm that uses an expression for this attribute.

Reply:
Added
https://github.com/wildfly/wildfly-core/blob/fd0416c917a0a599199f1c39d6a9dddf5a17080a/elytron/src/test/resources/org/wildfly/extension/elytron/elytron-expressions.xml#L105-L107